### ISSUE TITLE
ODV to DwC does not work with all Bio-ODV files?

### ISSUE THREAD

**Issue by LennertSchepers** [EMODnet/EMODnetBiocheck#24]
The conversion from ODV to DwC (ODVtoDwC function in https://github.com/EMODnet/EMODnetBiocheck/blob/master/R/ODVtoDwC.R) seems to have some problems with specific Bio-ODV files, such as the ones from https://www.seadatanet.org/Standards/Data-Transport-Formats.

**Comment by meliezer** [comment]
I confirm that the HTTP 500 error in the BioQC in https://vre.seadatanet.org/ doesn't happen with the original test file. I looked inside this file and it seems obvious. This service is NOT accepting SeaDataNet ODV spreadsheet format as used by SeaDataNet, but ODV spreadsheet format! SDN format does not have DataVariable. I converted one of the examples above to SeaDataNet ODV spreadsheet, but unfortunately I still get an error as a VRE user. I'll send the file by email.
